docs/guides/docs/intro-to-sway/explore-fuel.mdx

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I think we can remove this page, everything is linked in the nav already

Reply:
agreed. nuking it